File: NET/Libraries/Treehopper.Libraries/Sensors/Inertial/IAccelerometer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Numerics;
using System.Text;
using System.Threading.Tasks;

namespace Treehopper.Libraries.Sensors.Inertial
{
    /// <summary>
    /// Three-axis accelerometer sensor
    /// </summary>
    public interface IAccelerometer : IPollable
    {
        /// <summary>
        /// The three-axis acceleration reading, in g.
        /// </summary>
        Vector3 Accelerometer { get; }
    }
}

File: Java/api/treehopper.libraries/src/main/java/io/treehopper/libraries/sensors/inertial/IAccelerometer.java
package io.treehopper.libraries.sensors.inertial;

import com.badlogic.gdx.math.Vector3;
import io.treehopper.libraries.sensors.IPollable;

/**
 * Accelerometer interface
 */
public interface IAccelerometer extends IPollable {
    Vector3 getAccelerometer();
}
